modules/geographiclib/2.4.0.bcr.1/overlay/BUILD.bazel

```diff
@@ -0,0 +1,38 @@
+load(
+    "@cmake_configure_file//:cmake_configure_file.bzl",
+    "cmake_configure_file",
+)
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+package(default_visibility = ["//visibility:public"])
+
+cmake_configure_file(
+    name = "config_header",
+    src = "include/GeographicLib/Config.h.in",
+    out = "include/GeographicLib/Config.h",
+    cmakelists = [
+        "CMakeLists.txt",
+    ],
+    defines = [
+        "GEOGRAPHICLIB_DATA=/usr/local/share/GeographicLib",
+        "GEOGRAPHICLIB_HAVE_LONG_DOUBLE=0",
```

Comment:
Should this have the other variable substitutions here too? I.E.:

```
#define GEOGRAPHICLIB_VERSION_STRING "@PROJECT_VERSION@"
#define GEOGRAPHICLIB_VERSION_MAJOR @PROJECT_VERSION_MAJOR@
#define GEOGRAPHICLIB_VERSION_MINOR @PROJECT_VERSION_MINOR@
#define GEOGRAPHICLIB_VERSION_PATCH @PROJECT_VERSION_PATCH@
#define GEOGRAPHICLIB_PRECISION @GEOGRAPHICLIB_PRECISION@ 
```

Reply:
`cmake_configure_file `parse `set(VAR VALUE)` from CMakeLists.txt and set it automatically.
https://github.com/wep21/cmake_configure_file/blob/ff5077867f447c59bccb7b508d0ce5f385b9f0c5/cmake_configure_file.py#L123-L143

Reply:
oh cool! that's slick